src/main/java/com/gameList/gamelist/GameListController.java

```diff
@@ -54,14 +63,7 @@ private void loadData() {
 	
 		
 		// Nintendo
-		listNintendo = new ArrayList<Nintendo>();
-		Nintendo nintendoGame1 = new Nintendo(1, "Mario Party", "60.00", "N64");
-		Nintendo nintendoGame2 = new Nintendo(2, "Mario Party 2", "60.00", "N64");
-		Nintendo nintendoGame3 = new Nintendo(3, "Mario Party 3", "60.00", "N64");		
-		
-		listNintendo.add(nintendoGame1);
-		listNintendo.add(nintendoGame2);
-		listNintendo.add(nintendoGame3);
+		listNintendo = nintendoServiceStub.nintendoListCreator();
```

Comment:
I utilized a service stub (similar to what we learned from class) to make a list creator for your temporary mock data. This will make your controller look cleaner and not just full of mock data. I only did it for Nintendo list for now, but I would advise doing it for all of the models you have!